Library/Homebrew/cmd/info.rb

```diff
@@ -239,7 +239,7 @@ def info_formula(f, args:)
 
     if stable = f.stable
       s = "stable #{stable.version}"
-      s += " (bottled)" if stable.bottled?
+      s += " (bottled)" if stable.bottled? && (stable.bottle_specification.prefix == ENV["HOMEBREW_PREFIX"])
```

Comment:
```suggestion
      if f.bottled? && f.pour_bottle? && f.bottle_specification.compatible_cellar?
        s += " (bottled)" 
      end
```

which will also handle the missing case where `pour_bottle?` is `false` on your current system.

Reply:
Thanks, Mike. Your suggestion fixes the issue too. I believe `f.bottle_specification.compatible_cellar?` is checked by `f.bottled?`, so I'll update the PR accordingly
https://github.com/Homebrew/brew/blob/ff28d7c69b7b7e026757c63f15d00f98f07e5cec/Library/Homebrew/software_spec.rb#L94-L97